app/main.py

```diff
@@ -1 +1,31 @@
-# write your code here
+class Animal:
+    alive = []
+
+    def __init__(self, name: str):
+        self.name = name
+        self.health = 100
+        self.hidden = False
+        Animal.alive.append(self)
+
+    def __repr__(self):
+        return '{' + f"Name: {self.name}," \
```

Comment:
You can type '{' inside f-string by typing '{{'

Reply:
okay, thx